src/defaults.scss

```diff
@@ -0,0 +1,6 @@
+
+.cal-event {
+  background-color: #0062cc;
+  border-color: #005cbf;
+  color: #fff;
```

Comment:
How come this is needed?

Reply:
If there is nothing supplied it would use the primary color instead the one which fits to the background. That was my thought.

Reply:
If there is no color specified it will use the font set on a parent. It may doesn't fit to the background and needs extra Styling. That was my thought.